src/codeu/chat/server/Server.java

```diff
@@ -57,19 +61,30 @@
   private final Uuid id;
   private final Secret secret;
 
-  private final Model model = new Model();
-  private final View view = new View(model);
   private final Controller controller;
 
   private final Relay relay;
   private Uuid lastSeen = Uuid.NULL;
+  private final Model model;
+  final View view;
 
-  public Server(final Uuid id, final Secret secret, final Relay relay) {
+  public Server(final Uuid id, final Secret secret, final Relay relay, final File fp) {
 
```

Comment:
So server will extend file and then inside Server's constructor I will use the constructor `public File(URI uri)` to get the serialized data instead of adding another parameter to Server's constructor?